src/logger/LogBody.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default interface LogBody {
```

Comment:
This class should be a simple type instead of an interface. You just need to store four primitive data fields. There's no need for inheritance-based implementation.